torch/_dynamo/skipfiles.py

```diff
@@ -264,15 +307,15 @@ class SkipResult:
     reason: Optional[str]
 
 
-# TODO(ybliang): This is a temp function, we should consolidate this with check_verbose.
-def _check_verbose_inner(filename, allow_torch=False):
+# TODO(ybliang): This is a temp function, we should consolidate this with check_file.
+def _check_file_inner(filename, allow_torch=False):
     """Should skip this file?"""
     if filename is None:
         return SkipResult(True, "filename is None")
-    if filename in FILENAME_INLINELIST:
+    if any(filename.startswith(d) for d in get_file_inlinelist()):
```

Comment:
Are you sure about startswith? This seems to be different than the old behavior

Reply:
I think they are semantics equivalent.
Before this PR: Only file can be in ```FILENAME_INLINELIST```, so we do path exact match.
After this PR: Either file and directory can be in ```FILENAME_INLINELIST```.
* If it's a file, the only difference is we don't check the file name extensions. It should be ok since we only care about python file.
* If it's a directory, we are good to go with ```startswith``` since we want to inline all files under this directory. This is the same as adding all files under this directory and do path exact match.

Reply:
But these aren't files right, they're modules, no? So now you're going to start matching all submodules under the module, even if before you were only matching the top module

Reply:
Good point! But we are fine now, since ```torch/_export/db/examples/*py``` is the only one I lifted to ```torch._export.db.examples``` module startswith match, there is no other submodules under that folder, so we preserved the semantics.
But we should make the contract more clear in case anyone misuse it. If developers add a module/directory to this list, that means all sub files/directories will be added recursively. If they don't want to recursively add them, just add the files one by one. I'll update the docs.